packages/jasmine/src/coverage_runner.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2017 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+const childProcess = require('child_process');
+
+/**
+ * This is designed to collect the coverage of one target, since in nodejs
+ * and using NODE_V8_COVERAGE it may produce more than one coverage file, however bazel expects
+ * there to be only one lcov file. So this collects up the v8 coverage json's merges them and
+ * converts them to lcov for bazel to pick up later.
+ * TODO: The functionality in this script could be put into _lcov_merger then it would apply to
```

Comment:
😎Seems like we should land this functionality in the _lcov_merger binary with a nodejs_binary if possible. The trouble I see there is that that nodejs_binary would have a `@npm//c8` dependency then which would be a dep if coverage is enabled or not. Is that the blocker for that?

Reply:
You can make the dep on c8 dynamic like i've done with coverage_runner by just using `require` isntead of import. Then then the user gets an error they'd have to install c8 and provide it to the nodejs_binary, the same as how the current implementation works

Reply:
but I agree that nodejs_test is the right place to put this. I'd need your help to figure out the runfiles issues though

Reply:
I've added back the lcov_merger_js and added some debug info if you wanna take a look at the runfiels issues i've been having